tests/kvbm_integration/test_determinism_disagg.py

```
@@ -237,8 +236,8 @@ def _set_up_trtllm_config(self, gpu_cache_blocks):
             prefill_config_path,
             "--disaggregation-mode",
             "prefill",
-            "--connector",
-            "kvbm",
+            "--kv-transfer-config",
+            '{"kv_connector":"DynamoConnector","kv_connector_module_path":"kvbm.vllm_integration.connector","kv_role":"kv_both"}',
         ]
```

Ruff (0.15.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)